models.go

```diff
@@ -423,8 +425,8 @@ type RealmRepresentation struct {
 	BruteForceProtected                 bool              `json:"bruteForceProtected,omitempty"`
 	ClientAuthenticationFlow            string            `json:"clientAuthenticationFlow,omitempty"`
 	ClientScopeMappings                 map[string]string `json:"clientScopeMappings,omitempty"`
-	ClientScopes                        []interface{}     `json:"clientScopes,omitempty"`
-	Clients                             []interface{}     `json:"clients,omitempty"`
+	ClientScopes                        []ClientScope     `json:"clientScopes,omitempty"`
+	Clients                             []Client          `json:"clients,omitempty"`
```

Comment:
Its better to restric the use as required by the apis

Reply:
+1